provider/datagen/src/transform/cldr/time_zones/mod.rs

```diff
@@ -260,5 +263,26 @@ mod tests {
                 )
                 .unwrap()
         );
+
+        let metazone_period: DataPayload<MetaZonePeriodV1Marker> = provider
+            .load_resource(&DataRequest {
+                options: langid!("en").into(),
+                metadata: Default::default(),
+            })
+            .unwrap()
+            .take_payload()
+            .unwrap();
+        assert_eq!(
+            Some(MetaZoneId(tinystr!(4, "mgmt"))),
+            metazone_period
+                .get()
+                .0
+                .get(
+                    &TimeZoneBcp47Id(tinystr!(8, "gblon")),
+                    &String::from("1971-10-31 02:00")
+                )
+                .unwrap()
+                .get()
```

Comment:
An interesting I found is that 
```
            metazone_period
                .get()
                .0
                .get(
                    &TimeZoneBcp47Id(tinystr!(8, "gblon")),
                    &String::from("1971-10-31 02:00")
                )
                .unwrap()
``` 
becomes type `OptionULE<icu_datetime::provider::time_zones::MetaZoneId>`.

But what I defined in struct `MetaZonePeriodV1` is `Option<icu_datetime::provider::time_zones::MetaZoneId>`.

I did not use `OptionULE` while transforming the raw data. Did anything change the data type internally?

Reply:
Thanks!

but I didn't find method `get_copied` from https://unicode-org.github.io/icu4x-docs/doc/src/zerovec/ule/option.rs.html#26

Reply:
`get_copied` is on ZeroMap

https://unicode-org.github.io/icu4x-docs/doc/zerovec/struct.ZeroMap.html#method.get_copied